### PR TITLE
Refactor instance action resource and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Review by michaelkad — CHANGES_REQUESTED** [review]
Why is everything looks like they have been changed? I can tell what is the change here

File: ibm/service/power/resource_ibm_pi_instance_action.go [hunk 1]
```diff
@@ -37,44 +36,44 @@ func ResourceIBMPIInstanceAction() *schema.Resource {
...
-				Optional:     true,
-				ValidateFunc: validate.ValidateAllowedStringValues([]string{PVMInstanceHealthOk, PVMInstanceHealthWarning}),
 				Default:      PVMInstanceHealthOk,
 				Description:  "Set the health status of the PVM instance to connect it faster",
+				Optional:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validate.ValidateAllowedStringValues([]string{PVMInstanceHealthOk, PVMInstanceHealthWarning}),
 			},
 
 			// Computed
```
Comment (ismirlia) [inline ibm/service/power/resource_ibm_pi_instance_action.go:62]: Change computed to Attributes

Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_instance_action.go]: Changed

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIInstanceAction (363.41s)
PASS

**Comment by michaelkad** [comment]
Open #121

**Pull request closed without merging**